src/components/logs/TektonTaskRunLog.tsx

```diff
@@ -1,11 +1,12 @@
 import * as React from 'react';
+import { LogViewer } from '@patternfly/react-log-viewer';
 import { HttpError } from '@openshift-console/dynamic-plugin-sdk/lib/utils/error/http-error';
 import { TaskRunKind } from '../../types';
 import { TektonResourceLabel } from '../../consts';
-import './Logs.scss';
-import './MultiStreamLogs.scss';
 import { LoadingInline } from '../Loading';
 import { useTRTaskRunLog } from '../hooks/useTektonResult';
+import './Logs.scss';
+import './MultiStreamLogs.scss';
```

Comment:
Can we have all CSS for logs in one file now?

Reply:
Updated the PR. PTAL.